libraries/libfc/secp256k1/CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ add_library(secp256k1-internal INTERFACE)
 target_include_directories(secp256k1-internal INTERFACE secp256k1/src)
 
 target_compile_definitions(secp256k1-internal INTERFACE ENABLE_MODULE_RECOVERY=1
-                                                        ENABLE_MODULE_EXTRAKEYS=1
```

Comment:
`EXTRAKEYS` was added in https://github.com/AntelopeIO/spring/pull/142#discussion_r1602040845 and as I suspected at that time, this was indeed a dependency oversight and has been fixed in 0.5.1.